core/src/js/services/mercenaries/highlights/mercenaries-synergies-highlight-processor.ts

```diff
@@ -688,7 +701,7 @@ const worgen = (card: ReferenceCard) => race(card, Race.WORGEN);
 
 const alliance = or(draenei, dwarf, gnome, highelf, human, nightelf, worgen);
 const horde = or(bloodelf, goblin, halforc, orc, tauren, troll, undead);
-const explorer = (card: ReferenceCard) => hasMechanic(card, GameTag.MERCS_EXPLORER);
+const explorer = (card: ReferenceCard) => hasReferencedTag(card, GameTag.MERCS_EXPLORER);
```

Comment:
This one should really be `hasTag`, because we are looking for cards that are Explorers. Is that correct?

Reply:
Yes, this will need to be changed when you add "tags" field.